.github/workflows/build-package.yml

```diff
@@ -0,0 +1,61 @@
+name: Build Package
```

Comment:
can we give this a better name that also contains azure gdpr in it?

Reply:
Changed  and gave a more descriptive name